### PR TITLE
Global build browserify working

### PR DESCRIPTION
**Pull request by trusktr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "clean": "rimraf build",
     "lint": "eslint --ext js,jsx src test",
     "check": "npm run lint && npm run test:all",
-    "build": "babel src --out-dir build",
+    "build": "babel src --out-dir build && npm run build-global",
+    "build-global": "browserify -e build/index.js -s enzyme -x react/addons -x react/lib/ReactContext -x react/lib/ExecutionEnvironment -o build/global.js",
```
Comment (ljharb) [inline package.json]: Could we update this PR to add this command to the readme instead of adding the script?

File: package.json [hunk 2]
```diff
@@ -69,6 +70,8 @@
     "babel-loader": "^6.2.7",
     "babel-preset-airbnb": "^2.1.1",
     "babel-register": "^6.18.0",
+    "babelify": "^7.3.0",
```
Comment (ljharb) [inline package.json]: babelify doesn't seem used in the above command

**Comment by nikolalsvk** [comment]
When I run `npm run build` it fails with this error:
```
Error: Cannot find module 'react-test-renderer/shallow' from '/home/vagrant/enzyme/build'
    at /usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:111:15)
```

**Comment by ljharb** [comment]
@nikolalsvk with the React 15.5 changes, you now need to explicitly install `react-test-renderer` alongside enzyme.

**Comment by nikolalsvk** [comment]
@ljharb Thanks for responding 🙇  

I've installed `react-test-renderer` explicitly inside `enzyme` dir and tried to do `npm run build-global` and I'm now getting different kind of error:
```
Error: Cannot find module 'react-dom/test-utils' from '/home/vagrant/enzyme/build'
    at /home/vagrant/enzyme/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/home/vagrant/enzyme/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/home/vagrant/enzyme/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/home/vagrant/enzyme/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/home/vagrant/enzyme/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /home/vagrant/enzyme/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:111:15)
```
I'm doing this inside `enzyme` repo (airbnb/enzyme), is this a proper way to do it?

**Comment by ljharb** [comment]
Try `npm run react:15` to install the proper deps.

**Comment by ljharb** [comment]
Is this still relevant now that we're on v3?

**Comment by ljharb** [comment]
Closing; happy to reopen if you're interested in rebasing.

**Pull request closed without merging**